fabric-player-tracking-api-v1/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
I dont think this player tracking stuff belongs here? The module is not included in gradle?